ch1/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8" />
+    <title>My index page</title>
+  </head>
+
+  <body>
```

Comment:
```suggestion
  </head>
  <body>
```

주석으로 구분하는 등 특별한 경우가 아니면 태그 사이에 공백을 두지 않습니다.